steering_controllers_library/src/steering_odometry.cpp

```diff
@@ -181,30 +182,42 @@ void SteeringOdometry::set_odometry_type(const unsigned int type) { config_type_
 
 double SteeringOdometry::convert_twist_to_steering_angle(double v_bx, double omega_bz)
 {
-  if (omega_bz == 0 || v_bx == 0)
+  if (fabs(v_bx) < std::numeric_limits<float>::epsilon())
   {
-    return 0;
+    // avoid division by zero
+    return 0.;
   }
   return std::atan(omega_bz * wheelbase_ / v_bx);
 }
 
 std::tuple<std::vector<double>, std::vector<double>> SteeringOdometry::get_commands(
-  const double v_bx, const double omega_bz)
+  const double v_bx, const double omega_bz, const bool open_loop)
 {
   // desired wheel speed and steering angle of the middle of traction and steering axis
-  double Ws, phi;
+  double Ws, phi, phi_IK = steer_pos_;
 
+#if 0
```

Comment:
It was there from the beginning but wrong. I'd like to integrate the tricycle_controller (steering wheel = traction wheel), then this will be fixed in an upcoming PR.